components/HomePageForm.tsx

```diff
@@ -0,0 +1,103 @@
+const form = [
+    { label: "First name*", type: "text" },
+    { label: "Last name*", type: "text" },
+    { label: "E-mail*", type: "text" },
+    { label: "Company name*", type: "text" },
+    { label: "Website URL", type: "text" },
+]
+
+const select = [
+    { label: "Industry*", type: "text" },
+    { label: "Country*", type: "text" },
+    { label: "Employee count*", type: "text" },
+    { label: "Job title*", type: "text" }, 
+    { label: "Target start date*", type: "text" },
+    { label: "What kind of project is this?*", type: "text" },
+    { label: "Project budget*", type: "text" }, 
+]
+
+const selectForm = [
+    { label: "", text: ""},
+    { label: "1", text: "option 1"},
+    { label: "2", text: "option 2"},
+    { label: "3", text: "otpion 3"},
+]
+
+
+
+const handleChange = event => {
+    console.log(event.target.value);
+  }
+
+
+const HomePageForm = () => {
+    return(
```

Comment:
@Dzoni912 This looks good but we don't have values from input, we can insert it but how can we know which values are inserted? Lets make it, and on submit just show it in console.

Reply:
Yes, okay I understand what you mean. We will have that finished by tonight.